O365/sharepoint.py

```diff
@@ -99,7 +99,8 @@ class Site(ApiComponent):
 
     _endpoints = {
         'get_subsites': '/sites',
-        'get_lists': '/lists'
+        'get_lists': '/lists',
+        'get_listbyname':'/lists/displayName'
```

Comment:
As we are following PEP8, displayName should be display_name, and get_listbyname should be get_list_by_name.

Also, we are using format for string replacement. so instead of '/lists/display_name' use '/lists/{display_name}' so later you can format this path instead of using 'replace'

Reply:
Thankx , i have made the change 👍